assets/js/main.js

```diff
@@ -1,8 +1,13 @@
 import $ from 'jquery';
+import { accordion, accordionCloseButton, banner, navigation } from 'identity-style-guide';
```

Comment:
So happy about this. Instead of importing the entire js, only import what we need 👍